sql_generators/glean_usage/glean_app_ping_views.py

```diff
@@ -77,9 +78,11 @@ def generate_per_app(self, project_id, app_info, output_dir=None):
                     output_dir,
                     full_view_id,
                     "view.sql",
-                    VIEW_QUERY_TEMPLATE.format(
-                        full_view_id=full_view_id,
-                        target=underlying_view_id,
+                    reformat(
+                        VIEW_QUERY_TEMPLATE.format(
+                            full_view_id=full_view_id,
+                            target=underlying_view_id,
+                        )
```

Comment:
This is causing most of the changes in the diff. But necessary to make `verify-sql-format` happy